### ISSUE TITLE
bug

### ISSUE THREAD

**Issue by williamlzw** [fkodom/fft-conv-pytorch#6]
x=torch.randn(1,3,32,128)
net1=nn.Conv2d(3, 64, 1, (2,1), 1)
net2=FFTConv2d(3,64,1,(2,1),1)
out1=net1(x)
out2=net2(x)
print(out1.shape,out2.shape)
out!=out2

**Comment by fkodom** [comment]
Hi -- I'm not sure this is a bug?  The weights/biases for `nn.Conv2d` and `FFTConv2d` are randomly initialized, so in your example they will be different for the two layers.  Can you manually set the weights/biases to be equal, and see if the outputs are still different?

**Comment by williamlzw** [comment]
Not a bug, the padding and the stride switch places

**Comment by williamlzw** [comment]
nn.conv2d:
def __init__(
        self,
        in_channels,
        out_channels,
        kernel_size,
        stride,#<==
        padding,#<==
        dilation,
        groups,
        bias,
        padding_mode
    )

ffconv2d:
def __init__(
        self,
        in_channels,
        out_channels,
        kernel_size,
        padding,#<==
        stride,#<==
        dilation,
        groups
    )

**Comment by fkodom** [comment]
Understood.  I'll switch the positions of those arguments, to match the form of `f.conv2d`.

By the way, you can format code in Markdown using backticks:  https://www.freecodecamp.org/news/how-to-format-code-in-markdown/
Makes your code snippets much easier to read.
